### PR TITLE
fix: Add prefix to client's methods

### PR DESCRIPTION
**Pull request by mahatoankitkumar**
## Problem
When running client in jenkins it shows `multiple definition of `several client's functions as exp_client and cac_client' s functions had similar name.

## Solution
Added prefix to the extern function for both cac and exp clients as `cac_` and `exp_` respectively.

### CODE REVIEW

**Review by ayushjain17 — CHANGES_REQUESTED** [review]
update haskell client as well

**Review by ayushjain17 — APPROVED** [review]

**Review by sauraww — APPROVED** [review]

File: headers/libexperimentation_client.h [hunk 1]
```diff
@@ -5,22 +5,22 @@
 
 typedef struct Arc_Client Arc_Client;
 
-int last_error_length(void);
+int exp_last_error_length(void);
```
Comment (knutties) [inline headers/libexperimentation_client.h:8]: @mahatoankitkumar - can we use `expt` as the prefix for experimentation - that seems a little more standard and is also a little less ambiguous.

https://www.merriam-webster.com/dictionary/expt

**Review by knutties — CHANGES_REQUESTED** [review]
Please consider changing the experimentation prefix to `expt`.

File: headers/libexperimentation_client.h (hunk 1 above)
Comment (mahatoankitkumar) [reply headers/libexperimentation_client.h]: Yeah it's fine, but https://www.merriam-webster.com/dictionary/exp.

Comment (knutties) [reply headers/libexperimentation_client.h]: The ambiguity is explicit in that page 😄

**Review by Datron — APPROVED** [review]

**Review by knutties — APPROVED** [review]

**Pull request merged**